SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,85 @@
+Supported Operations
+####################
+
+.. table::
+
```

Comment:
Could you add a small heading and under that says "High-level tensor types" and under that add a small sentence. Something like this. 

**High-Level Tensors**
All the protocols use these high-level tensors for their operations with some operations supported by these tensors. 

[The Table]